lib/recent_test.go

```diff
@@ -0,0 +1,89 @@
+package lib
+
+import (
+	"github.com/stretchr/testify/assert"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+func Test_convertData(t *testing.T) {
+	recentFileContent, err := os.ReadFile("../test-data/recent/RECENT_OLD_FORMAT")
+	if err != nil {
+		logger.Error("Could not open file ../test-data/recent/RECENT_OLD_FORMAT")
+	}
+
+	var recentFileData RecentFiles
+	convertData(recentFileContent, &recentFileData)
+	assert.Equal(t, "1.5.6", recentFileData.Terraform[0])
```

Comment:
Probably worth asserting the length of `recentFileData.Terraform` before referencing slice indexes

Reply:
Done